package.json

```diff
@@ -5,12 +5,12 @@
     "doc": "docs"
   },
   "engines": {
-    "node": "20.9.0",
-    "pnpm": "8.10.0"
+    "node": "^20.9.0",
+    "pnpm": "^8.10.0"
```

Comment:
That's not really a change that should be part of this PR scope, but I am fine with it.